vita3k/gui/src/game_install_dialog.cpp

```diff
@@ -110,10 +116,15 @@ void draw_game_install_dialog(GuiState &gui, HostState &host) {
     if (ImGui::BeginPopupModal("Game installation failed", nullptr, ImGuiWindowFlags_AlwaysAutoResize)) {
         ImGui::PopStyleColor();
         ImGui::TextColored(GUI_COLOR_TEXT, "Installation failed, please check the log for more information.");
+        ImGui::Checkbox("Delete Game file installer?", &delete_game_file);
```

Comment:
Same message in two places in the same method? 🤔 

Just curious. No change requested.

Reply:
Is for install succès and install failed so need write same 2 time